pdsfile/pdsfile.py

```diff
@@ -1296,7 +1296,10 @@ def os_path_exists(cls, abspath, force_case_sensitive=False):
                 if cls.os_path_exists(shelf_abspath + '_info.pickle'):
                     return True
 
-                # Checksum files need special handling
+                # Checksum files need special handling, before doing special handling,
+                # check if the checksums directory exist
+                if 'checksums-' in abspath and not os.path.exists(abspath):
```

Comment:
I think it is the wrong thing to do in these functions to check the filesystem if the shelve file doesn't exist. Instead we should change OPUS to not require shelves for PDS4 until such time as we have them available. I've made that change. I think these changes should be reverted to still require the shelve files.